XcodeServerSDK/Server Entities/Issue.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Issue.swift
+//  XcodeServerSDK
+//
+//  Created by Mateusz Zając on 04.08.2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+public class Issue: XcodeServerEntity {
+    
+    public enum Type: String {
+        case BuildServiceError = "buildServiceError"
+        // Looks like build service warnings are treated as warnings in case
+        // of type. In case of Issue Type they're Build Service Error
+        // case BuildServiceWarining = "warning"
+        case TriggerError = "triggerError"
+        case Error = "error"
+        case Warning = "warning"
+        case TestFailure = "testFailure"
+        case AnalyzerWarning = "analyzerWarning"
```

Comment:
How it's even possible? 😮

Here's my JSON for `buildServiceWarnings`:
```json
"buildServiceWarnings": [
{
  "status": 0,
  "age": 0,
  "message": "No such file or directory - old dSYM file cannot be overwritten:\t\told: '/Library/Developer/XcodeServer/Integrations/Caches/c40f3384eed2ea14647e2236a117d0a8/DerivedData/Build/Products/Debug-iphonesimulator/HeatChat.app.dSYM/Contents/Resources/DWARF/HeatChat'",
  "commits": [],
  "integrationID": "12993d83910012e7cb7fc57023069f3b",
  "type": "warning",
  "issueType": "Build Service Error"
},
{
  "status": 0,
  "age": 0,
  "message": "Skipping exporting from archive because creating the archive failed.",
  "commits": [],
  "integrationID": "12993d83910012e7cb7fc57023069f3b",
  "type": "warning",
  "issueType": "Build Service Error"
}
]
```